pkg/alertmanager/alerts/alerts.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
```

Comment:
>> Where does the new protobuf come in?
> I chose to use a protobuf for the polling interface boundary to provide a easy to use storage format that wasn't yaml based and easy to version. This would make any additional backend based on cloud provider object stores much easier to add, while not reducing any of the current functionality.

Could you re-explain to me (which I'm slow to understand :)) why we do need protobuf given the current use case? Let's say, without considering what the future could hold for us...

Reply:
It will allow for more robust serialization and versioning. This will ensure a stable format for communicating alertmanager configs.